src/drivers/boards/px4fmu-v4pro/board_config.h

```diff
@@ -330,7 +328,7 @@ __BEGIN_DECLS
 #define BOARD_ADC_BRICK2_VALID  (px4_arch_gpioread(GPIO_VDD_BRICK2_VALID))
 #define BOARD_ADC_SERVO_VALID   (1)
 #define BOARD_ADC_PERIPH_5V_OC  (px4_arch_gpioread(GPIO_VDD_5V_PERIPH_OC))
-#define BOARD_ADC_HIPOWER_5V_OC (0)
+#define BOARD_ADC_HIPOWER_5V_OC (px4_arch_gpioread(GPIO_VDD_5V_HIPOWER_OC))
```

Comment:
I made the change, it makes sense. For the moment both 
`BOARD_ADC_HIPOWER_5V_OC ` and `BOARD_ADC_PERIPH_5V_OC ` are not used in [commander](https://github.com/PX4/Firmware/blob/master/src/modules/commander/commander.cpp#L1967-L2000), they appear only in [sdlog2](https://github.com/PX4/Firmware/blob/master/src/modules/sdlog2/sdlog2.c#L2099-L2115) for logging purposes. We should also add logic for the second brick, would that be better in a different PR?